README.rst

```diff
@@ -154,6 +149,117 @@ Usage
    the changes made to it will be visible for both the Docker
    container and the Windows system.
 
+#.  Check the shared folder :
+
+    - run command
+
+    .. code:: sh
+
+	ls
+
+    - the docker_shared_folder should be highlighted in green : 
+	.. image:: /picture/screenshot_green.PNG
+	
+      if not : 
+	- check the permission on your local host file :
+	    - go to C:/Users/
+	    - right click on the folder open properties of the folder.
+	    - go to the security tab 
+		
+	    - Check that the USER have full control over the folder.
+
+	    .. image:: /picture/permission1.png
+
+	    - if yes move on to next paragraph.
+
+	    - if not change the permission and run the ls command again in the docker quickstart terminal. 
+	    - if the folder is highlighted in green try creating a folder inside it.
+
+	    .. code:: sh
+
+		mkdir test
+
+	    - Check if a new folder appeared in C:/Users/docker_folder_shared
+	
+		
+	- Go to the docker quickstart terminal
+
+		- Stop Docker Machine if it's running, by running command:
```

Comment:
Indented by too much, should be aligned with G

Reply:
You mean - Stop Docker Machine if it's running, by running command: 
should be aligned with G ?